### PR TITLE
fix reuse stoneslab Paper recipe

### PR DESCRIPTION
**Pull request by MotiasF33**

### CODE REVIEW

**Comment by Dream-Master** [comment]
@MotiasF33 can you change that in gt code?

File: scripts/Minecraft.zs [hunk 1]
```diff
@@ -2398,6 +2403,12 @@ recipes.addShaped(<minecraft:fishing_rod>, [
 [null, <ore:stickLongWood>, <minecraft:string>],
 [<ore:stickLongWood>, <ore:craftingToolWireCutter>, <ore:ringAnyIron>]]);
 
+// --- Paper
+recipes.addShaped(<minecraft:paper> * 2, [
+[null, <minecraft:stone_slab>.reuse(), null],
+[<ore:dustPaper>, <ore:dustPaper>, <ore:dustPaper>],
+[null, <minecraft:stone_slab>.reuse(), null]]);
+
```
Comment (Dream-Master) [inline scripts/Minecraft.zs:2411]: Put this change in gt code please

**Review by Dream-Master — CHANGES_REQUESTED** [review]

File: scripts/Minecraft.zs (hunk 1 above)
Comment (MotiasF33) [reply scripts/Minecraft.zs:2411]: @Dream-Master , you mean a gt.zs ?

Comment (Prometheus0000) [reply scripts/Minecraft.zs]: As part of https://github.com/GTNewHorizons/GT5-Unofficial

**Comment by Dream-Master** [comment]
nope i said no mt script recipe

File: scripts/Minecraft.zs (hunk 1 above)
Comment (MotiasF33) [reply scripts/Minecraft.zs]: @Prometheus0000, I understood that code GT, but where exactly?

Comment (MotiasF33) [reply scripts/Minecraft.zs]: I would be glad if you can add it 
or tell me where to add it 😽

**Comment by MotiasF33** [comment]
> 
> 
> nope i said no mt script recipe

I am confused about how to add reusability to an item in a recipe.. 🤔

**Pull request closed without merging**